app/controllers/application_controller.rb

```diff
@@ -1,2 +1,23 @@
 class ApplicationController < ActionController::Base
+  def sign_out
```

Comment:
This doesn't feel like it belongs in `ApplicationController`, particularly as some controllers that inherit from this don't have any authentication. How about moving this to `Staff::SignoutController` or similiar?

Reply:
I don't think they belong in `Auth0Controller` as they log users out of the application as well as Auth0, but I'll see if a `Staff::` controller works

Reply:
There's definitely something odd happening here, so I'm accepting this for now!